src/views/Home.vue

```diff
@@ -1,5 +1,11 @@
 <template>
   <div class="v-home flex min-h-full">
+    <div
+      class="np-product-added-modal"
+      :class="productAddedMessage ? 'np-product-added-modal--in' : ''"
```

Comment:
in case of single class binding you can use `:class="{'np-product-added-modal--in': productAddedMessage}"` syntax